index.js

```diff
@@ -1,102 +0,0 @@
-const readline = require("readline");
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-start();
-
-async function start() {
-  console.log("Choose your game!!");
-  let game = "false";
-
-  function ask(questionText) {
-    return new Promise((resolve, reject) => {
-      rl.question(questionText, resolve);
-    });
-  }
-
-  game = await ask("reverse or guess? ");
-  if (game === "guess") {
-    let minNumber = 1;
-//Original Start
-    let minCurrent = 0;
-    ("yes");
-    while (playAgain === "yes") {
```

Comment:
Initially the game errors here. On the first run of the game the player is not presented with the "play again" choice. So you will want to do a different/or additional check here to account for that.